langchain/chains/loading.py

```diff
@@ -352,6 +352,9 @@ def _load_sql_database_chain(config: dict, **kwargs: Any) -> SQLDatabaseChain:
         llm = load_llm_from_config(llm_config)
     elif "llm_path" in config:
         llm = load_llm(config.pop("llm_path"))
+    elif "llm_chain" in config:
```

Comment:
i think if llmchain is present, we want to load the whole chain and then pass that in. so NOT pop the llm and use the `from_llm` constructor

Reply:
ok, i'll refactor this

Reply:
i notice that when load from string, a secretMap param can be passed, but when load from a file, we don't have this params, is it designed for that? @hwchase17